android/build.gradle

```diff
@@ -41,6 +45,7 @@ android {
         targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
 
         buildConfigField "Long", "AsyncStorage_db_size", "${dbSizeInMB}L"
+        buildConfigField("boolean", "AsyncStorage_use_AsyncTaskThreadPoolExecutor", "${useAsyncTaskThreadPoolExecutor}")
```

Comment:
nit: could we rename it to something less typo-prone name? Like `AsyncStorage_customExecutor` ?